src/utilities/workflowAnnotationUtils.ts

```diff
@@ -31,19 +24,15 @@ export function getWorkflowAnnotations(
       helmChartPaths: deploymentConfig.helmChartFilePaths,
       provider: 'GitHub'
    }
-   const prefixedAnnotationObject = prefixObjectKeys(
-      annotationObject,
-      ANNOTATION_PREFIX
-   )
-   return JSON.stringify(prefixedAnnotationObject)
+   return JSON.stringify(annotationObject)
 }
 
 export function getWorkflowAnnotationKeyLabel(
    workflowFilePath: string
 ): string {
    const hashKey = require('crypto')
       .createHash('MD5')
-      .update(`${process.env.GITHUB_REPOSITORY}/${workflowFilePath}`)
+      .update(`${ANNOTATION_PREFIX}/${workflowFilePath}`)
       .digest('hex')
    return `githubWorkflow_${hashKey}`
```

Comment:
let's move the hashkey around here. For this function just return `${ANNOTATION_PREFIX}/k8s-deploy`. That means we can remove the workflowFilePath parameter.